### DISCUSSION TITLE
how do landing page and other pages relate?

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#97] [Q&A] [answered]
Landing page is a really powerful feature of doks theme, but there might be a more simple case for documentation where one just wants to get to `https://getdoks.org/docs/prologue/introduction/` as a starting page. What would allow to skip `index.html`?

So far my understanding is that https://github.com/h-enk/doks/blob/master/content/_index.md is extended with 
https://github.com/h-enk/doks/blob/master/layouts/index.html, but what should one do to skip it? Does this involve heavy redesign?

**Comment by h-enk** [comment]
You could just:

1. Rename `.layouts/index.html` to `.layouts/index.bak.html` (see also: [ Examples: Layout Lookup for Home Page](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-home-page)), and
2. Add to the front matter of the page you would like to be the homepage: `url: /` (see also [ Front Matter](https://gohugo.io/content-management/front-matter))

**Reply by epogrebnyak** [reply]
Thank you so much for the idea - it was really not trivial that a front matter can control the root url, I was not even thinking that direction.

**Comment by epogrebnyak** [comment, marked as answer]
I took a route of copying ```\layouts\docs\single.html``` as ```\layouts\index.html``` which allows to reflect the left-side TOC on the first page, also commented out secondary TOC.

Another thing I tried before this was to assign `url="/"` in front matter - but when you do that to some document that is on a tree like 
```content/docs/prologue/introduction.md``` - that document is forced into othre URL and will not show as ```content/docs/prologue/introduction``` (404 error). That makes one think a landing page must still be ```content\_index.md``` controlled by ```\layouts\index.html```. Hope this is a good path. )

![изображение](https://user-images.githubusercontent.com/9265326/106491619-1dbb8f00-64c8-11eb-8c1e-d225197d9a80.png)
